packages/components/src/VirtualizedList/ListTable/ListTable.scss

```
@@ -23,9 +23,8 @@ $tc-list-table-border: 1px solid darken($concrete, 5%);
 		}
 	}
 
-	.tc-list-item {
+	:global(.ReactVirtualized__Table__headerRow) {
```

Limit global usage to where it's necessary, please
```suggestion
	.tc-list-headerRow {
```